README.md

```diff
@@ -41,6 +26,12 @@ If you pull down commits with updates to the schema make sure you run migrations
 yarn prisma migrate dev
 ```
 
+Sometimes you might also want to generate Typescript types, perhaps after a schema.prisma change
+
+```sh
+yarn prisma generate
+```
+
```

Comment:
The `yarn setup` above should also run `prisma generate` (along with migrations and seed) so I think this unnecessary.

Reply:
yeah this might be in the wrong section, its more of a general dev tip